Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -8693,6 +8749,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the custom name visiblity of this monster. If it's false, you only see the name when you sight the mob. If it's true, you always see the custom name.",
 				},
+				SetLeashedTo =
+				{
+					Params =
+					{
+						{
+							Name = "Entity",
+							Type = "cEntity",
+						},
+					},
+					Notes = "Leash the monster to a entity.",
+				},
+				SetUnleashed =
+				{
+					Params =
+					{
+						{
+							Name = "ShouldDropPickup",
+							Type = "boolean",
+						},
+					},
+					Notes = "Unleash the monster from an entity.",
```

Comment:
Those functions should be "on the monster" which is being leashed / unleashed, not "on the entity" to which the monster is being leashed.

I think the Lua API would be best if it could be used like this:
```lua
-- monsterX are cMonster instances from somewhere
monster1:LeashTo(player1)
monster2:LeashToPos(Vector3i(blockX, blockY, blockZ))  -- creates a new leash knot
monster3:Unleash()
```